### PR TITLE
fix: stop culling when taking avatar snapshot in avatar editor

### PR DESCRIPTION
**Pull request by pbosio**

### CODE REVIEW

File: unity-client/Assets/Rendering/Culling/CullingController.cs [hunk 1]
```diff
@@ -330,17 +331,17 @@ internal void ResetObjects()
             var renderers = objectsTracker.GetRenderers();
             var animations = objectsTracker.GetAnimations();
 
-            for (var i = 0; i < skinnedRenderers.Length; i++)
+            for (var i = 0; i < skinnedRenderers?.Length; i++)
```
Comment (sandrade-dcl) [inline unity-client/Assets/Rendering/Culling/CullingController.cs:334]: Only for curiossity: why did you need to add the `?` operator? Were we receiving sometimes the `skinnedRenderers` as null and it maked the function flow fail?

**Review by sandrade-dcl — APPROVED** [review]
LGTM!

File: unity-client/Assets/Rendering/Culling/CullingController.cs (hunk 1 above)
Comment (pbosio) [reply unity-client/Assets/Rendering/Culling/CullingController.cs:334]: I don't think those array can be null in runtime, but there were throwing an annoying error when stopping unity editor :)

**Review by AjimenezDCL — APPROVED** [review]

**Review by pravusjif — APPROVED** [review]
Tested snapshots with a guest user with no problems.
Also saw no problems dealing with other rendered avatars around me.
